TrkHitReco/fcl/prolog.fcl

```diff
@@ -13,8 +13,8 @@ makeSH : {
   MaximumTime             : 1690.0 # ns
   MinimumEnergy           : 0.0001 # MeV
   MaximumEnergy           : 0.0035 # MeV
-  MinimumRadius           : 0.0 # mm
-  MaximumRadius           : 800.0 # mm
+  MinimumRadius           : 395.0 # mm
+  MaximumRadius           : 650.0 # mm
```

Comment:
I would change the setting of the same parameters in the `makePH` module instance to avoid confusion. 
Also, I think that maybe a de-activation of X straws at smaller(larger) radial distance w.r.t. the solenoid axis would allow to obtain a more uniform selection (I'm not asking to make this change in this PR, but I wanted to mention it to keep it in mind for the future)